assets/src/block-editor/helpers/index.js

```diff
@@ -413,13 +413,19 @@ AmpLayoutControl.propTypes = {
 /**
  * Get AMP Noloading toggle control.
  *
+ * @deprecated As of v2.1. Blocks with the `ampNoLoading` attribute will still be able to use the control.
+ *
  * @param {Object} props Props.
  *
  * @return {ReactElement} Element.
  */
 const AmpNoloadingToggle = ( props ) => {
 	const { attributes: { ampNoLoading }, setAttributes } = props;
 
+	if ( ! ampNoLoading ) {
```

Comment:
Should this be:

```suggestion
	if ( undefined === ampNoLoading ) {
```

If it's just `! ampNoLoading`, then the toggle will be hidden on existing blocks that have it set to false. And if an existing block has it set to true and it is toggled to false, the toggle will disappear. That's not what we intend, right?

Reply:
Yea that's true, nice catch. I've addressed that in https://github.com/ampproject/amp-wp/pull/5574/commits/b3909733647e9e8a0096bf9e39816b81e94667f0.